js/scripts/three-class-config.js

```diff
@@ -845,7 +862,7 @@ module.exports = {
             encoding:         new Types.Enum('TextureEncodings', 'LinearEncoding'),
             version:          new Types.Int(0),
         },
-        construtorArgs: [ 'image', 'mapping', 'wrapS', 'wrapT', 'magFilter', 'minFilter', 'format', 'type', 'anisotropy' ],
+        //constructorArgs: [ 'image', 'mapping', 'wrapS', 'wrapT', 'magFilter', 'minFilter', 'format', 'type', 'anisotropy' ],
```

Comment:
I put the comment in the commit message, as I needed input on what we want here. The gist of it is basically:
 - This initially had a typo (missing a 'c').
 - Fixing the typo breaks the class, as there is no `image` property defined.

Should we expose the `image` property? If not, does it make sense to define the constructor args, as they will necessarily be different than the three docs.